### DISCUSSION TITLE
Sushi de Pobre

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#371] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/sushi-de-pobre/

**Comment by brunoaugusto** [comment]
*Sanjuuichi Hasado*. Essa foi genial!

**Comment by tali-zorah** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/sushi-de-pobre/#comment_1087512636>):

> *Sanjuuichi Hasado*. Essa foi genial!

*Keelah*... Demorei tanto pra chegar nesse nome

**Comment by najarin-the-loremaster** [comment]
> [Tali'Zorah nar Rayya](</perfil/?user=tali-zorah>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/sushi-de-pobre/#comment_1087513601>):

> *Keelah*... Demorei tanto pra chegar nesse nome

.Ficou excelente! É como diz o ditado: "O único lugar onde sucesso vem antes de trabalho é no dicionário" — pelo menos nos idiomas que eu consigo ler &gt;.&lt;
